### PR TITLE
FOUR-12709: FOUR-11533: Add Metric "Request of process"

### PR DESCRIPTION
**Pull request by pmPaulis**
## Issue & Reproduction Steps
FOUR-11533: Add Metric "Request of process"

## Solution
- List the changes you've introduced to solve the issue.

## How to Test
Describe how to test that this solution works.

## Related Tickets & Packages
- https://processmaker.atlassian.net/browse/FOUR-12709

## Code Review Checklist
- [ ] I have pulled this code locally and tested it on my instance, along with any associated packages.
- [ ] This code adheres to [ProcessMaker Coding Guidelines](https://github.com/ProcessMaker/processmaker/wiki/Coding-Guidelines).
- [ ] This code includes a unit test or an E2E test that tests its functionality, or is covered by an existing test.
- [ ] This solution fixes the bug reported in the original ticket.
- [ ] This solution does not alter the expected output of a component in a way that would break existing Processes.
- [ ] This solution does not implement any breaking changes that would invalidate documentation or cause existing Processes to fail.
- [ ] This solution has been tested with enterprise packages that rely on its functionality and does not introduce bugs in those packages.
- [ ] This code does not duplicate functionality that already exists in the framework or in ProcessMaker.
- [ ] This ticket conforms to the PRD associated with this part of ProcessMaker.
ci:deploy
ci:next

### CODE REVIEW

**Comment by tokensPM** [comment]
QA server was successfully deployed https://ci-1744f930eb.eng.processmaker.net

**Comment by tokensPM** [comment]
QA server was successfully deployed https://ci-1744f930eb.eng.processmaker.net

**Comment by tokensPM** [comment]
QA server was successfully deployed https://ci-1744f930eb.eng.processmaker.net

File: resources/js/processes-catalogue/components/optionsMenu/ProcessCounter.vue [hunk 1]
```diff
@@ -1,5 +1,64 @@
...
+  padding-top: 0;
+  width: 294px;
+  height: 52px;
+  border-radius: 8px;
+  border: 1px solid #CDDDEE;
+  background: #FFF;
+}
+.process-counter-text {
+  color: #556271;
+  font-family: "Open Sans", sans-serif;
```
Comment (fagubla) [inline resources/js/processes-catalogue/components/optionsMenu/ProcessCounter.vue:48]: remove this line
this font is by default in all PM4

File: resources/js/processes-catalogue/components/optionsMenu/ProcessCounter.vue [hunk 2]
```diff
@@ -1,5 +1,64 @@
...
+  font-size: 16px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 24px;
+  letter-spacing: -0.32px;
+}
+.process-counter-total {
+  color: #556271;
+  font-family: "Open Sans", sans-serif;
```
Comment (fagubla) [inline resources/js/processes-catalogue/components/optionsMenu/ProcessCounter.vue:57]: remove this line
this font is by default in all PM4

**Review by fagubla — CHANGES_REQUESTED** [review]

**Review by fagubla — APPROVED** [review]

**Comment by tokensPM** [comment]
QA server was successfully deployed https://ci-1744f930eb.eng.processmaker.net

**Pull request merged**